src/sendResponse.js

```diff
@@ -0,0 +1,11 @@
+const sendResponse = (res, response, statusCode, data) => {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = statusCode;
+  res.statusText = response;
+
+  res.end(data);
```

Comment:
JSON.stringify should be used to convert data to a JSON string before sending it in the response.